LearningMissionLab/ScheduleEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class ScheduleEvent
+    {
+        DateTime _dayOfTheWeek;
+        DateTime _startTimeHour;
+        DateTime _startTimeMinutes;
```

Comment:
startTimeMinutes should be int or uint. When you set the StartTimeMinutes property, you should be able to restrict the value to 59.